lib/rules/cli/4.1.yaml

```diff
@@ -166,6 +166,7 @@
   :options:
     :clusterrole: --clusterrole=<value>
     :group: --group=<value>
+    :serviceaccount: --serviceaccount=<value>
```

Comment:
We also need to add it to 4.4.yaml
```
$ ls -l lib/rules/cli/4.*
-rw-rw-r--. 1 lxia lxia 37983 Mar 12 10:20 lib/rules/cli/4.1.yaml
lrwxrwxrwx. 1 lxia lxia     8 Dec  5 15:59 lib/rules/cli/4.2.yaml -> 4.1.yaml
lrwxrwxrwx. 1 lxia lxia     8 Dec  5 15:59 lib/rules/cli/4.3.yaml -> 4.2.yaml
-rw-rw-r--. 1 lxia lxia 37987 Mar 12 10:20 lib/rules/cli/4.4.yaml
lrwxrwxrwx. 1 lxia lxia     8 Mar 11 17:04 lib/rules/cli/4.5.yaml -> 4.4.yaml
```

Reply:
@liangxia I have added the same option in 4.4.yaml, please check. Thanks!